tests/sdk/testTransfer.py

```diff
@@ -44,6 +45,15 @@ def test_success(self):
         transfer = starkbank.transfer.get(id=transfer_id)
         self.assertIsNotNone(transfer.id)
         self.assertEqual(transfer.id, transfer_id)
+    
+    def test_success_ids(self):
+        transfers = starkbank.transfer.query(limit=5)
+        transfers_ids_expected = [t.id for t in transfers]
+        transfers_ids_result = [t.id for t in starkbank.transfer.query(ids=transfers_ids_expected)]
+        transfers_ids_expected.sort()
+        transfers_ids_result.sort()
+        self.assertIsNotNone(transfers_ids_result)
```

Comment:
Me parece que transactions_ids_result pode ser uma lista vazia, mas não None

Reply:
Changed to assertTrue.